modules/tensor_mechanics/include/userobjects/TensorMechanicsPlasticWeakPlaneTensileN.h

```diff
@@ -29,55 +29,17 @@ class TensorMechanicsPlasticWeakPlaneTensileN : public TensorMechanicsPlasticWea
   virtual std::string modelName() const;
 
  protected:
+  Real yieldFunction(const RankTwoTensor & stress, Real intnl) const override;
 
-  /**
-   * The yield function
-   * @param stress the stress at which to calculate the yield function
-   * @param intnl internal parameter
-   * @return the yield function
-   */
-  Real yieldFunction(const RankTwoTensor & stress, Real intnl) const;
-
-  /**
-   * The derivative of yield function with respect to stress
-   * @param stress the stress at which to calculate the yield function
-   * @param intnl internal parameter
-   * @return df_dstress(i, j) = dyieldFunction/dstress(i, j)
-   */
-  RankTwoTensor dyieldFunction_dstress(const RankTwoTensor & stress, Real intnl) const;
-
-  /**
-   * The derivative of yield function with respect to the internal parameter
-   * @param stress the stress at which to calculate the yield function
-   * @param intnl internal parameter
-   * @return the derivative
-   */
-  Real dyieldFunction_dintnl(const RankTwoTensor & stress, Real intnl) const;
-
-  /**
-   * The flow potential
-   * @param stress the stress at which to calculate the flow potential
-   * @param intnl internal parameter
-   * @return the flow potential
-   */
-  RankTwoTensor flowPotential(const RankTwoTensor & stress, Real intnl) const;
-
-  /**
-   * The derivative of the flow potential with respect to stress
-   * @param stress the stress at which to calculate the flow potential
-   * @param intnl internal parameter
-   * @return dr_dstress(i, j, k, l) = dr(i, j)/dstress(k, l)
-   */
-  RankFourTensor dflowPotential_dstress(const RankTwoTensor & stress, Real intnl) const;
-
-  /**
-   * The derivative of the flow potential with respect to the internal parameter
-   * @param stress the stress at which to calculate the flow potential
-   * @param intnl internal parameter
-   * @return dr_dintnl(i, j) = dr(i, j)/dintnl
-   */
-  RankTwoTensor dflowPotential_dintnl(const RankTwoTensor & stress, Real intnl) const;
+  RankTwoTensor dyieldFunction_dstress(const RankTwoTensor & stress, Real intnl) const override;
 
+  Real dyieldFunction_dintnl(const RankTwoTensor & stress, Real intnl) const override;
+
+  RankTwoTensor flowPotential(const RankTwoTensor & stress, Real intnl) const override;
+
+  RankFourTensor dflowPotential_dstress(const RankTwoTensor & stress, Real intnl) const override;
+
+  RankTwoTensor dflowPotential_dintnl(const RankTwoTensor & stress, Real intnl) const override;
```

Comment:
Nice! I appreciate that you didn't add documentation to these overridden methods.